rustle-my-net/src/main.rs

```diff
@@ -1,3 +1,31 @@
+mod input;
+mod structs;
+extern crate clap;
+use clap::{Arg, App, SubCommand};
+use input::read;
+use std::env;
 fn main() {
-    println!("Hello, world!");
+    let matches = App::new("rustle my net")
+        .subcommand(SubCommand::with_name("learn"))
+        .subcommand(SubCommand::with_name("classify"))
+        .arg(Arg::with_name("data")
+             .long("data")
+             .short("d")
+             .takes_value(true)
+             .default_value("data/iris_flowers.txt"))
+        .arg(Arg::with_name("config")
+             .long("config")
+             .short("c")
+             .takes_value(true)
+             .default_value("data/iris_flowers.txt"))
+        .get_matches();
```

Comment:
I think we should put all this parsing stuff into it's own module tbh, at least not into main(). Maybe as well into input/mod.rs